crates/autopilot/src/run_loop.rs

```diff
@@ -297,14 +298,15 @@ impl RunLoop {
                     &driver_,
                     solution_id,
                     solved_order_uids,
+                    solver,
                     auction_id,
                     block_deadline,
                 )
                 .await
             {
                 Ok(tx_hash) => {
                     Metrics::settle_ok(&driver_, submission_start.elapsed());
-                    tracing::debug!(?tx_hash, driver = %driver_.name, "solution settled");
+                    tracing::debug!(?tx_hash, ?solver, driver = %driver_.name, "solution settled");
```

Comment:
nit: can we have the `solver` arg last? That way the more important information shows up earlier in the logs which makes it slightly nicer to work with. Not blocking.

Reply:
fixed